Fix hyphen-used-as-minus-sign and manpage-has-errors-from-man
Index: mc-4.7.0-pre1/doc/man/ru/mc.1.in
===================================================================
--- mc-4.7.0-pre1.orig/doc/man/ru/mc.1.in	2009-08-02 22:54:19.000000000 +0200
+++ mc-4.7.0-pre1/doc/man/ru/mc.1.in	2009-08-02 22:58:52.000000000 +0200
@@ -3,18 +3,18 @@
 .TH MC 1 "Январь 2003" "MC Version 4.6.0" "GNU Midnight Commander"
 .\"SKIP_SECTION"
 .SH "НАИМЕНОВАНИЕ"
-mc \- Визуальная оболочка для Unix-подобных систем.
+mc \- Визуальная оболочка для Unix\-подобных систем.
 .\"SKIP_SECTION"
 .SH "СИНТАКСИС"
 .B mc
-[\-abcCdfhPstuUVx] [\-l log] [dir1 [dir2]] [-v file] [-e [файл]]
+[\-abcCdfhPstuUVx] [\-l log] [dir1 [dir2]] [\-v file] [\-e [файл]]
 .\"NODE "DESCRIPTION"
 .SH "ОПИСАНИЕ"
 Что такое Midnight Commander
 .PP
-Midnight Commander - это программа, предназначенная для просмотра
+Midnight Commander \- это программа, предназначенная для просмотра
 содержимого каталогов и выполнения основных функций управления файлами в
-UNIX-подобных операционных системах.
+UNIX\-подобных операционных системах.
 .\"NODE "OPTIONS"
 .\".\"DONT_SPLIT"
 .SH "КЛЮЧИ"
@@ -24,7 +24,7 @@
 Отключить использование графических символов для рисования линий.
 .TP
 .I \-b
-Запуск программы в черно-белом режиме экрана.
+Запуск программы в черно\-белом режиме экрана.
 .TP
 .I \-c
 Включает цветной режим дисплея. Для получения более подробной информации
@@ -48,7 +48,7 @@
 .\"LINK2"
 виртуальной файловой системы smbfs
 .\"SMB File System"
-для установки уровня отладочных сообщений smbfs в N (0-10).
+для установки уровня отладочных сообщений smbfs в N (0\-10).
 .TP
 .I \-e [файл]
 Запустить встроенный редактор.  Если параметр файл задан, этот файл
@@ -66,7 +66,7 @@
 функциональные клавиши не работают.
 .TP
 .I \-l файл
-Сохранить диалог с ftp-сервером, а также отладочную информацию smbfs в
+Сохранить диалог с ftp\-сервером, а также отладочную информацию smbfs в
 файл file.
 .TP
 .I \-P файл
@@ -105,7 +105,7 @@
 как опции).
 .TP
 .I \-v file
-Запустить встроенную программу просмотра Midnight Commander-а для
+Запустить встроенную программу просмотра Midnight Commander\-а для
 просмотра указанного файла. После выхода из режима просмотра вы выходите
 из Midnight Commander и оказываетесь в shell.
 .TP
@@ -119,20 +119,20 @@
 modes, and able to send mouse escape sequences).
 .PP
 Первое путевое имя (path name) в командной строке (если указано) задает
-каталог, который будет отображаться в активной панели; второй путь -
+каталог, который будет отображаться в активной панели; второй путь \-
 каталог, отображаемый во второй панели.
 .\"NODE "Overview"
 .SH "Главное окно программы"
 Главное окно программы Midnight Commander состоит из трех полей. Два
 поля, называемые "панелями", идентичны по структуре и обычно отображают
-перечни файлов и подкаталогов каких-то двух каталогов файловой
+перечни файлов и подкаталогов каких\-то двух каталогов файловой
 структуры. Эти каталоги в общем случае различны, хотя, в частности,
 могут и совпасть. Каждая панель состоит из заголовка, списка файлов и
 информационной строки.
 .PP
 Третье поле экрана, расположенное в нижней части экрана, содержит
 командную строку текущей оболочки. В этом же поле (самая нижняя строка
-экрана) содержится подсказка по использованию функциональных клавиш F1 -
+экрана) содержится подсказка по использованию функциональных клавиш F1 \-
 F10. Самая верхняя строка экрана содержит строку
 .\"LINK2"
 горизонтального меню\&.
@@ -147,7 +147,7 @@
 каталогов или файлов, а также выделен цветом заголовок панели в верхней
 строке. Этот заголовок совпадает с именем отображаемого в данной панели
 каталога, который является текущим каталогом той оболочки, из которой
-запущена программа. Вторая панель - пассивна. Почти все операции
+запущена программа. Вторая панель \- пассивна. Почти все операции
 выполняются в активной панели, то есть в соответствующем (текущем)
 каталоге. Некоторые операции (типа копирования или переноса файлов) по
 умолчанию используют каталог, отображаемый в пассивной панели, как место
@@ -240,16 +240,16 @@
 (иногда обозначается ALT или даже Compose). В настоящем руководстве
 будут использоваться следующие сокращения:
 .PP
-C-<символ> означает, что нужно нажать и удерживать клавишу Control и в
-это время нажать клавишу <символ>. Так C-f будет означать: нажмите
+C\-<символ> означает, что нужно нажать и удерживать клавишу Control и в
+это время нажать клавишу <символ>. Так C\-f будет означать: нажмите
 Control и, удерживая ее, нажмите (коротким щелчком) клавишу f.
 .PP
-Аналогично M-<символ> означает, что надо удерживать в нажатом состоянии
+Аналогично M\-<символ> означает, что надо удерживать в нажатом состоянии
 клавишу Meta или Alt во время удара по клавише <символ>. Если на
 клавиатуре нет клавиш Meta и Alt, нажмите ESC, отпустите ее, а потом
 щелкните по клавише <символ>.
 .PP
-S-<символ> означает, что нужно держать в нажатом состоянии клавишу Shift
+S\-<символ> означает, что нужно держать в нажатом состоянии клавишу Shift
 во время удара по клавише <символ>.
 .PP
 Все строки ввода в Midnight Commander поддерживают соглашения об
@@ -298,7 +298,7 @@
 клавиш, которые не попадают ни в одну из других категорий:
 .PP
 .BR Enter .
-Если в командной строке (в нижней части экрана) набрано имя какой-либо
+Если в командной строке (в нижней части экрана) набрано имя какой\-либо
 команды, запускается соответствующая программа (если команды не
 существует, выдается соответствующее сообщение). Если в командной строке
 нет никакого текста, и если на активной панели подсвечено имя каталога,
@@ -313,78 +313,78 @@
 .\"Extension File Edit"
 то выполняется команда, связанная с этим расширением.
 .TP
-.B C-l.
+.B C\-l.
 Обновляет всю информацию, отображаемую программой Midnight Commander.
 .TP
-.B C-x c.
+.B C\-x c.
 Запускает команду
 .\"LINK2"
 Chmod
 .\"Chmod"
 для подсвеченного файла или группы отмеченных файлов.
 .TP
-.B C-x o.
+.B C\-x o.
 Запускает команду
 .\"LINK2"
 Chown
 .\"Chown"
 для подсвеченного файла или группы отмеченных файлов.
 .TP
-.B C-x l.
+.B C\-x l.
 Запускает команду создания жесткой ссылки.
 .TP
-.B C-x s.
+.B C\-x s.
 Запускает команду создания символической ссылки.
 .TP
-.B C-x i.
+.B C\-x i.
 Переводит пассивную панель в режим "Информация".
 .TP
-.B C-x q.
+.B C\-x q.
 Переводит пассивную панель в режим быстрого просмотра.
 .TP
-.B C-x !.
+.B C\-x !.
 Выполняется пункт меню
 .\"LINK2"
 "Критерий панелизации"\&.
 .\"External panelize"
 .TP
-.B C-x h
+.B C\-x h
 Добавить имя каталога в
 .\"LINK2"
 "Справочник каталогов"\&.
 .\"Hotlist"
 .TP
-.B M-!,
+.B M\-!,
 Выполняет пункт меню "Просмотр вывода команды", который описан подробнее
 в разделе
 .\"LINK2"
 "Меню "Файл"\&.
 .\"File Menu"
 .TP
-.B M-?,
+.B M\-?,
 Вызов диалогового окна
 .\"LINK2"
 "Поиск файла"\&.
 .\"Find File"
 .TP
-.B M-c,
+.B M\-c,
 Вызов диалогового окна
 .\"LINK2"
 "Быстрая смена каталога"\&.
 .\"Quick cd"
 .TP
-.B C-o,
+.B C\-o,
 Если Midnight Commander запущен в консоли Linux или FreeBSD, или через
 xterm, вам будет показан вывод предыдущей команды. При этом на
-Linux-консоли Midnight Commander использует встроенную программу
+Linux\-консоли Midnight Commander использует встроенную программу
 (cons.saver) для сохранения выводимой информации и демонстрации ее на
 экране.
 .PP
 Если Midnight Commander скомпилирован с поддержкой подоболочки
-(subshell), вы можете в процессе выполнения приложения из-под Midnight
-Commander в любой момент набрать C-o и вернуться к главному экрану
-Midnight Commander-а. Для возврата к вашему приложению достаточно снова
-набрать C-o. Если вы застопорите выполнение приложения, использовав этот
+(subshell), вы можете в процессе выполнения приложения из\-под Midnight
+Commander в любой момент набрать C\-o и вернуться к главному экрану
+Midnight Commander\-а. Для возврата к вашему приложению достаточно снова
+набрать C\-o. Если вы застопорите выполнение приложения, использовав этот
 прием, вы не сможете запустить других программ из Midnight Commander
 пока отложенное приложение не закончит работу, либо пока вы не прервете
 его выполнение.
@@ -397,7 +397,7 @@
 Меню левой и правой панелей\&.
 .\"Left and Right Menus"
 .PP
-.B Tab, C-i.
+.B Tab, C\-i.
 Сменить текущую (активную) панель. Подсветка перемещается с панели,
 которая была активной ранее, в другую панель, которая становится
 активной.
@@ -407,40 +407,40 @@
 используйте клавишу Insert (the kich1 terminfo sequence). Для снятия 
 отметки с файла используются те же комбинации.
 .PP
-.B C-t.
-Чтобы поменять кодировку панели, используйте комбинацию C-t (Control-t).
+.B C\-t.
+Чтобы поменять кодировку панели, используйте комбинацию C\-t (Control\-t).
 Перекодировка производится из выбранной кодировки в системную. Для отмены
 перекодировки просто перейдите на каталог вверх '..'. Для отмены
 перекодировки всех каталогов выберите "Без перекодировки" в диалоге
 выбора кодировок.
 .PP
-.B M-g, M-r, M-j.
+.B M\-g, M\-r, M\-j.
 Используются для перемещения подсветки, соответственно, на самый
 верхний, средний или нижний файл из числа отображаемых в данный момент
 на панели.
 .PP
-.B C-s, M-s.
+.B C\-s, M\-s.
 Инициирует режим поиска имен файлов в текущем каталоге по первым
 символам имени. После нажатия одной из этих комбинаций, вводимые символы
 отображаются не в командной строке, а в строке поиска. Если режим
-.I "Показ мини-статуса"
-включен, эта строка отображается на месте строки мини-статуса. В
+.I "Показ мини\-статуса"
+включен, эта строка отображается на месте строки мини\-статуса. В
 процессе ввода последующих символов линия подсветки перемещается к
 следующему файлу, название которого начинается с введенной строки
 символов. Клавиши
 .I backspace
 или
 .I DEL
-могут использоваться для исправления ошибок. Если C-s нажато снова,
+могут использоваться для исправления ошибок. Если C\-s нажато снова,
 осуществляется поиск следующего подходящего файла.
 .PP
-.B M-t
+.B M\-t
 Циклически переключает режимы отображения списка файлов текущего
 каталога. С помощью этой комбинации клавиш можно быстро переключаться из
 режима стандартного вывода (long listing) к сокращенному или к режиму,
 определяемому пользователем.
 .PP
-.B C-\\\\ (control-backslash).
+.B C\-\\\\ (control\-backslash).
 Показать
 .\"LINK2"
 Справочник каталогов
@@ -472,44 +472,44 @@
 Клавиша "\\" снимает отметку с группы файлов, то есть производит
 действие, обратное тому, которое вызывается по клавише "+".
 .PP
-.B up-key, C-p.
+.B up\-key, C\-p.
 Перемещает подсветку на предыдущую позицию в списке файлов панели.
 .PP
-.B down-key, C-n.
+.B down\-key, C\-n.
 Перемещает подсветку на следующую позицию в списке файлов панели.
 .PP
-.B home, a1, M-<.
+.B home, a1, M\-<.
 Перемещает подсветку на первую позицию списка файлов.
 .PP
-.B end, c1, M->.
+.B end, c1, M\->.
 Перемещает подсветку на последнюю позицию списка файлов.
 .PP
-.B next-page, C-v.
+.B next\-page, C\-v.
 Перемещает подсветку на одну страницу вниз.
 .PP
-.B prev-page, M-v.
+.B prev\-page, M\-v.
 Перемещает подсветку на одну страницу вверх.
 .PP
-.B M-o,
+.B M\-o,
 Сделать текущий каталог активной панели также текущим каталогом
 неактивной панели.  Если необходимо, перевести неактивную панель в режим
 отображения списка файлов.  Если активная панель находится в режиме
 панелизации, неактивная панель не переводится в этот режим.
 .PP
-.B C-PageUp, C-PageDown
+.B C\-PageUp, C\-PageDown
 Только если mc запущен с консоли Linux: выполняется, соответственно,
 переход (chdir) к родительскому каталогу ("..") или к выделенному
 подсветкой каталогу.
 .PP
-.B M-y
+.B M\-y
 Перемещение к предыдущему каталогу из истории перемещения по каталогам;
 эквивалентно нажатию мышкой на символ '<' в верхнем углу панели.
 .PP
-.B M-u
+.B M\-u
 Перемещение к следующему каталогу из истории перемещения по каталогам;
 эквивалентно нажатию мышкой на символ '>'.
 .PP
-.B M-S-h, M-H
+.B M\-S\-h, M\-H
 Отображает историю перемещения по каталогам; эквивалентно нажатию мышкой
 на символ 'v' в верхнем углу панели.
 .\"NODE "  Shell Command Line"
@@ -518,13 +518,13 @@
 нажатий на клавиши во время ввода и редактирования команд в командной
 строке.
 .PP
-.BR M-Enter .
+.BR M\-Enter .
 Копирует подсвеченное имя файла или каталога в командную строку.
 .PP
-.BR C-Enter .
-То же самое, что M-Enter, но работает только на консоли Linux.
+.BR C\-Enter .
+То же самое, что M\-Enter, но работает только на консоли Linux.
 .PP
-.BR M-Tab .
+.BR M\-Tab .
 Пытается выполнить операцию
 .\"LINK2"
 Завершение ввода (completion)
@@ -533,27 +533,27 @@
 машины (в зависимости от того, что вы начали набирать и какой элемент
 команды вводите).
 .PP
-.B C-x t, C-x C-t.
+.B C\-x t, C\-x C\-t.
 Копирует в командную строку имена помеченных файлов (или подсвеченное
-имя, если нет помеченных) из активной панели (C-x t) или пассивной
-панели (C-x C-T).
+имя, если нет помеченных) из активной панели (C\-x t) или пассивной
+панели (C\-x C\-T).
 .PP
-.B C-x p, C-x C-p.
+.B C\-x p, C\-x C\-p.
 Первая комбинация клавиш копирует в командную строку имя текущего
-каталога, а вторая - имя каталога, отображаемого в пассивной панели.
+каталога, а вторая \- имя каталога, отображаемого в пассивной панели.
 .PP
-.BR C-q .
+.BR C\-q .
 Эта команда (the quote command) используется для того, чтобы вставить
-символы, которые каким-то образом интерпретируются самим Midnight
-Commander-ом (например, символ '+').
+символы, которые каким\-то образом интерпретируются самим Midnight
+Commander\-ом (например, символ '+').
 .PP
-.B M-p, M-n.
-Эти комбинации используются для перемещения по истории команд. M-p
+.B M\-p, M\-n.
+Эти комбинации используются для перемещения по истории команд. M\-p
 вызывает перемещение на команду назад по списку ранее запускавшихся
-команд, а M-n - перемещение на одну команду вперед.
+команд, а M\-n \- перемещение на одну команду вперед.
 .PP
-.BR M-h .
-Выводит историю текущей строки ввода (для командной строки - историю
+.BR M\-h .
+Выводит историю текущей строки ввода (для командной строки \- историю
 команд).
 .\"NODE "  General Movement Keys"
 .SH "  Клавиши управления перемещением"
@@ -563,20 +563,20 @@
 одни и те же комбинации клавиш. Но в каждой подпрограмме имеются и
 комбинации, применяющиеся только в ней.
 .PP
-Другие части Midnight Commander-а тоже используют некоторые из
+Другие части Midnight Commander\-а тоже используют некоторые из
 комбинаций клавиш управления перемещением, так что настоящая секция
 руководства может быть также полезна при изучении этих частей.
 .PP
-.B Up, C-p.
+.B Up, C\-p.
 Перемещение на одну строку назад или вверх
 .PP
-.B Down, C-n.
+.B Down, C\-n.
 Перемещение на одну строку вперед
 .PP
-.B Prev Page, Page Up, M-v.
+.B Prev Page, Page Up, M\-v.
 Перемещение на одну страницу назад
 .PP
-.B Next Page, Page Down, C-v.
+.B Next Page, Page Down, C\-v.
 Перемещение на одну страницу вперед
 .PP
 .B Home, A1.
@@ -588,7 +588,7 @@
 Программы просмотра файлов и помощи воспринимают в дополнение к
 перечисленным еще и следующие комбинации:
 .PP
-.B b, C-b, C-h, Backspace, Delete.
+.B b, C\-b, C\-h, Backspace, Delete.
 Перемещение на одну страницу назад
 .PP
 .B Space bar.
@@ -608,56 +608,56 @@
 оболочки, но и строки в диалоговых окнах различных подпрограмм)
 воспринимают следующие управляющие комбинации:
 .PP
-.B C-a.
+.B C\-a.
 Перемещает курсор в началу строки.
 .PP
-.B C-e.
+.B C\-e.
 Перемещает курсор в конец строки.
 .PP
-.B C-b, стрелка-влево.
+.B C\-b, стрелка\-влево.
 Перемещает курсор на одну позицию влево.
 .PP
-.B C-f, стрелка-вправо.
+.B C\-f, стрелка\-вправо.
 Перемещает курсор на одну позицию вправо.
 .PP
-.B M-f.
+.B M\-f.
 Перемещает курсор на одно слово вперед.
 .PP
-.B M-b.
+.B M\-b.
 Перемещает курсор на одно слово назад.
 .PP
-.B C-h, backspace.
+.B C\-h, backspace.
 Удаляет символ, предшествующий курсору.
 .PP
-.B C-d, Delete.
+.B C\-d, Delete.
 Удаляет символ в позиции курсора.
 .PP
-.B C-@.
+.B C\-@.
 Устанавливает метку для того, чтобы вырезать (скопировать в буфер) часть
 текста.
 .PP
-.B C-w.
+.B C\-w.
 Копирует текст, расположенный между курсором и меткой, в буфер, удаляя
 текст из строки ввода.
 .PP
-.B M-w.
+.B M\-w.
 Копирует текст, расположенный между курсором и меткой,
 в буфер.
 .PP
-.B C-y.
+.B C\-y.
 Вставляет содержимое буфера в строку ввода перед позицией курсора.
 .PP
-.B C-k.
+.B C\-k.
 Удаляет текст от курсора до конца строки.
 .PP
-.B M-p, M-n.
-Эти комбинации используются для перемещения по истории команд. M-p
-перемещает к предыдущей команде, M-n - к следующей.
+.B M\-p, M\-n.
+Эти комбинации используются для перемещения по истории команд. M\-p
+перемещает к предыдущей команде, M\-n \- к следующей.
 .PP
-.B M-C-h, M-Backspace.
+.B M\-C\-h, M\-Backspace.
 Удалить предшествующее слово.
 .PP
-.BR M-Tab .
+.BR M\-Tab .
 Пытается выполнить
 .\"LINK2"
 завершение ввода (completion)
@@ -684,7 +684,7 @@
 .\"LINK2"
 "Файл"
 .\"File Menu"
-позволяет выполнить какие-то действия с выбранным файлом или группой
+позволяет выполнить какие\-то действия с выбранным файлом или группой
 помеченных файлов.
 .PP
 Меню
@@ -727,7 +727,7 @@
 .PP
 В расширенном (long) формате содержимое каталога представляется так, как
 это делает команда
-.BR "ls -l" .
+.BR "ls \-l" .
 В этом формате панель занимает весь экран.
 .PP
 Если вы выберете формат "Определяемый пользователем" ("User"), вы должны
@@ -750,25 +750,25 @@
 .TP
 .B bsize
 отображать размер в альтернативной форме, при которой выводятся размеры
-файлов, а для каталогов выводится только надпись "ПОД-КАТ" ("SUB-DIR")
-или "ВВЕРХ" ("UP--DIR").
+файлов, а для каталогов выводится только надпись "ПОД\-КАТ" ("SUB\-DIR")
+или "ВВЕРХ" ("UP\-\-DIR").
 .TP
 .B type
 отображать односимвольное поле типа. Этот символ может принимать
 значения из подмножества символов, выводимых командой
-.BR "ls -F" :
+.BR "ls \-F" :
 .B *
 (asterisk) для исполняемых файлов,
 .B /
 (slash) для каталогов,
 .B @
-(at-sign) для ссылок (links),
+(at\-sign) для ссылок (links),
 .B =
 (знак равенства) для сокетов (sockets),
-.B -
-дефис (hyphen) для байт-ориентированных устройств,
+.B \-
+дефис (hyphen) для байт\-ориентированных устройств,
 .B +
-для блок-ориентированных устройств,
+для блок\-ориентированных устройств,
 .B |
 (pipe) для файлов типа FIFO,
 .B ~
@@ -778,7 +778,7 @@
 (ссылок, указывающих в никуда).
 .TP
 .B mark
-Звездочка (asterisk) если файл помечен, пробел - если не помечен.
+Звездочка (asterisk) если файл помечен, пробел \- если не помечен.
 .TP
 .B mtime
 время последней модификации файла.
@@ -874,14 +874,14 @@
 Отображение списка файлов в любой из панелей может производиться в
 соответствии с одним из восьми порядков сортировки:
 .nf
-- по имени;
-- по расширению;
-- по размеру файла;
-- по времени модификации;
-- по времени последнего обращения к файлу;
-- по времени последнего изменения атрибутов файла;
-- по номеру узла (inode);
-- без сортировки.
+\- по имени;
+\- по расширению;
+\- по размеру файла;
+\- по времени модификации;
+\- по времени последнего обращения к файлу;
+\- по времени последнего изменения атрибутов файла;
+\- по номеру узла (inode);
+\- без сортировки.
 .fi
 .PP
 Порядок сортировки можно выбрать, вызвав диалоговое окно, в котором
@@ -916,17 +916,17 @@
 .\"External panelize"
 .\"NODE "  File Menu"
 .SH "  Файл"
-Программа Midnight Commander использует функциональные клавиши F1 - F10
+Программа Midnight Commander использует функциональные клавиши F1 \- F10
 как "горячие" клавиши для команд, включенных в меню "Файл".
-Escape-последовательности, генерируемые клавишами F1-F10, соответствуют
-функциям terminfo kf1 - kf10. На терминалах без поддержки функциональных
+Escape\-последовательности, генерируемые клавишами F1\-F10, соответствуют
+функциям terminfo kf1 \- kf10. На терминалах без поддержки функциональных
 клавиш можно достичь аналогичного эффекта, нажав клавишу ESC, а
-затем число в диапазоне от 1 до 9 или 0 (соответствует F1 - F9 и F10).
+затем число в диапазоне от 1 до 9 или 0 (соответствует F1 \- F9 и F10).
 .PP
 Меню "Файл" содержит следующие команды (соответствующие "горячие"
 клавиши указываются в скобках):
 .PP
-.B Просмотр файла (F3, Shift-F3)
+.B Просмотр файла (F3, Shift\-F3)
 .PP
 Просмотреть файл, на который указывает подсветка. По умолчанию при этом
 вызывается
@@ -941,10 +941,10 @@
 Если переменная
 .B PAGER
 не определена, вызывается встроенная программа. Если вместо F3 нажать
-Shift-F3, встроенная программа просмотра не выполняет предварительного
+Shift\-F3, встроенная программа просмотра не выполняет предварительного
 форматирования или обработки файла.
 .PP
-.B Просмотр вывода команды (Filtered View) (M-!)
+.B Просмотр вывода команды (Filtered View) (M\-!)
 .PP
 По этой команде на экране появляется строка ввода, в которой вы можете
 ввести любую команду с параметрами (по умолчанию предлагается
@@ -983,7 +983,7 @@
 .\"Mask Copy/Rename"
 .PP
 В процессе выполнения копирования можно нажать
-.B C-c
+.B C\-c
 или
 .B ESC
 для того, чтобы прервать выполнение операции.
@@ -991,22 +991,22 @@
 В некоторых системах можно осуществлять копирование в фоновом режиме,
 если запустить процесс не кнопкой <Дальше> диалогового окна, а кнопкой
 <На фоне> (или нажав
-.B M-b
+.B M\-b
 в диалоговом окне).  Используя пункт меню
 .\"LINK2"
 "Команды/Фоновые задания"\&,
 .\"Background Jobs"
 можно управлять выполнением фоновых процессов.
 .PP
-.B Права доступа (C-x c)
+.B Права доступа (C\-x c)
 .PP
 Позволяет изменить права доступа к выделенному или помеченным файлам.
 .PP
-.B Жесткая ссылка (C-x l)
+.B Жесткая ссылка (C\-x l)
 .PP
 Создает жесткую ссылку на текущий файл.
 .PP
-.B Символич. ссылка (C-x s)
+.B Символич. ссылка (C\-x s)
 .PP
 Создает символическую ссылку на текущий файл. Если вы не знаете, что
 такое ссылки: создание ссылки в некотором смысле подобно копированию
@@ -1018,23 +1018,23 @@
 Жесткая ссылка выглядит как реальный файл. После создания жесткой ссылки
 невозможно различить, где исходный файл, а где ссылка. Если вы удаляете
 один из этих файлов (точнее одно из этих имен), то файл еще сохраняется
-на диске (пока у него есть хоть одно имя-ссылка). Очень трудно различить
+на диске (пока у него есть хоть одно имя\-ссылка). Очень трудно различить
 первоначальное имя файла и позже созданные жесткие ссылки на него.
 Поэтому жесткие ссылки применяются там, где отслеживать различия и не
 требуется.
 .PP
-Символическая ссылка - это ссылка на имя исходного файла. Если исходный
+Символическая ссылка \- это ссылка на имя исходного файла. Если исходный
 файл удален, символическая ссылка становится бесполезной. Символическую
 ссылку легко отличить от первоначального имени файла и программа
 Midnight Commander указывает символические ссылки выводя знак "@" перед
 именем такой ссылки (кроме ссылок на каталоги, которые обозначаются
-знаком тильды "~"). Если на экран выводится строка мини-статуса (опция
-"Показывать мини-статус" ("Show mini-status") включена), то в ней
+знаком тильды "~"). Если на экран выводится строка мини\-статуса (опция
+"Показывать мини\-статус" ("Show mini\-status") включена), то в ней
 отображается имя исходного файла. Используйте символические ссылки в тех
 случаях, когда хотите избежать путаницы, связанной с применением жестких
 ссылок.
 .PP
-.B Владелец/группа (C-x o)
+.B Владелец/группа (C\-x o)
 Позволяет выполнить команду chown.
 .PP
 .B Права (расширенные)
@@ -1060,12 +1060,12 @@
 .PP
 Удаляется файл, имя которого подсвечено (или группа файлов, имена
 которых помечены) в активной панели. Операцию можно прервать, нажав
-.B C-c
+.B C\-c
 или
 .B ESC
 во время ее исполнения.
 .PP
-.B Быстрая смена каталога (Quick cd) (M-c)
+.B Быстрая смена каталога (Quick cd) (M\-c)
 .\"    Quick cd"
 .PP
 Используйте
@@ -1097,16 +1097,16 @@
 .I С учётом регистра
 отключена, то регистр символов учитываться не будет.
 .PP
-.B Снять отметку (\\\\ - обратный слэш)
+.B Снять отметку (\\\\ \- обратный слэш)
 .PP
 Используется для снятия отметки с группы файлов. Эта операция является
 обратной по отношению к операции отметки группы файлов и использует те
 же правила формирования шаблонов.
 .PP
-.B Выход (F10, Shift-F10)
+.B Выход (F10, Shift\-F10)
 .PP
-Выйти из программы Midnight Commander. Shift-F10 используется для выхода
-в тех ситуациях, когда используется shell wrapper. После Shift-F10
+Выйти из программы Midnight Commander. Shift\-F10 используется для выхода
+в тех ситуациях, когда используется shell wrapper. После Shift\-F10
 текущим каталогом будет не последний каталог, в котором вы были при
 работе в Midnight Commander, а тот, в котором вы были, когда запустили
 Midnight Commander.
@@ -1148,25 +1148,25 @@
 .I Поиск файла
 .\"Find File"
 позволяет найти на диске файл с заданным именем. "Горячие" клавиши \-
-.BR M-? .
+.BR M\-? .
 .PP
 Команда
 .I "Переставить панели"
 ("Swap panels") меняет местами содержимое
 правой и левой панелей. "Горячие" клавиши \-
-.BR C-u .
+.BR C\-u .
 .PP
 По команде
 .I "Отключить панели"
 ("Panels on/off") показывается вывод последней из выполнявшихся команд
 shell. Эта команда работает только через xterm и на консоли Linux и
 FreeBSD. "Горячие" клавиши \-
-.BR C-o .
+.BR C\-o .
 .PP
 По команде
 .I "Сравнить каталоги"
 ("Compare directories", "горячие" клавиши \-
-.BR "C-x d" )
+.BR "C\-x d" )
 сравнивается содержимое каталогов, отображаемых на левой и правой
 панелях. После сравнения можно воспользоваться командой
 .I "Копировать" (F5)
@@ -1183,37 +1183,37 @@
 .I "Критерий панелизации"
 .\"External panelize"
 позволяет выполнить внешнюю программу, сделав ее вывод содержимым
-текущей панели (характерный пример - панелизация вывода команды find).
+текущей панели (характерный пример \- панелизация вывода команды find).
 .PP
 .PP
 Пункт меню
 .I "История команд"
 выводит окно со списком ранее выполнявшихся
 команд. Подсвеченную строку из истории можно скопировать в командную
-строку оболочки (перемещение подсветки - стрелками вверх/вниз,
-копирование - по клавише Enter). Доступ к истории команд можно получить
+строку оболочки (перемещение подсветки \- стрелками вверх/вниз,
+копирование \- по клавише Enter). Доступ к истории команд можно получить
 также по комбинациям клавиш
-.B M-p
+.B M\-p
 или
-.BR M-n ,
+.BR M\-n ,
 однако в этом случае вы не видите окна с перечнем команд. Вместо этого в
 командную строку выводится одна команда из списка и по комбинации
-.B M-p
+.B M\-p
 происходит смена этой команды на предыдущую, а по
-.B M-n
+.B M\-n
 \- на следующую команду из истории команд.
 .PP
 Пункт меню
 .\"LINK2"
 .I "Быстрый доступ к каталогам"
 .\"Hotlist"
-.B C-\\\\
+.B C\-\\\\
 позволяет создать список каталогов, которые часто используются, и
 обеспечить быстрый переход к нужному каталогу из этого списка.
 .PP
 Пункт меню
 .I "Список активных ВФС"
-.B C-x a
+.B C\-x a
 отображает список виртуальных файловых систем, активных на настоящий
 момент. Можно перейти на выбранную виртуальную файловую систему. Для
 Обновления списка выберите кнопку "Освежить" в диалоговом окне.
@@ -1224,7 +1224,7 @@
 .\"LINK2"
 .I "Фоновые задания"
 .\"Background Jobs"
-.B C-x j
+.B C\-x j
 отображает список фоновых заданий, запущенных в диалоге 
 копирования/перемещения кнопкой "В фоне".
 .PP
@@ -1257,7 +1257,7 @@
 Midnight Commander создает дерево путем просмотра только небольшого
 подмножества всех каталогов. Если каталог, который вам нужен, не
 отображен, перейдите в его родительский каталог и нажмите
-.B C-r
+.B C\-r
 (или
 .BR F2 ).
 .PP
@@ -1276,7 +1276,7 @@
 нажатии клавиши Enter соответствующий каталог отображается во второй
 панели, а на текущей панели остается дерево.
 .PP
-.B C-r, F2 (Rescan).
+.B C\-r, F2 (Rescan).
 Перечитать содержимое каталога. Используется в тех случаях, когда дерево
 каталогов не соответствует реальной структуре: некоторые каталоги не
 показаны или показаны более не существующие.
@@ -1318,12 +1318,12 @@
 .B F8 (Delete).
 Удалить подсвеченный каталог из файловой системы.
 .PP
-.B C-s, M-s.
+.B C\-s, M\-s.
 Найти следующий каталог, соответствующий заданному шаблону поиска. Если
 такого каталога не существует, происходит просто смещение подсветки на
 одну строку вниз.
 .PP
-.B C-h, Backspace.
+.B C\-h, Backspace.
 Удаляет последний символ в строке (шаблоне) поиска.
 .PP
 .B Любой другой символ.
@@ -1331,8 +1331,8 @@
 имя следующего каталога, удовлетворяющее шаблону. В режиме просмотра
 дерева каталогов вначале необходимо активизировать режим поиска
 клавишами
-.BR C-s .
-Шаблон поиска будет отображаться в строке мини-статуса.
+.BR C\-s .
+Шаблон поиска будет отображаться в строке мини\-статуса.
 .PP
 Следующие действия возможны только в окне просмотра дерева каталогов и
 не поддерживаются при просмотре дерева в одной из панелей:
@@ -1375,10 +1375,10 @@
 поиска. Кнопка <Выход> (<Quit>) служит для выхода из режима поиска.
 Нажатие на кнопку <Панелизация> (<Panelize>) приведет к тому, что
 результаты поиска будут отображены на текущую активную панель, так что
-вы можете производить с выбранными файлами еще какие-то действия
+вы можете производить с выбранными файлами еще какие\-то действия
 (просматривать, копировать, перемещать, удалять и так далее). После
 вывода на панель можно нажать
-.B C-r
+.B C\-r
 для возврата к обычному списку файлов.
 .PP
 Можно задать список каталогов, которые команда "Поиск файла" будет
@@ -1412,12 +1412,12 @@
 "Перенаправление вывода на панель") позволяет выполнить внешнюю
 программу и отобразить вывод этой программы на текущей активной панели.
 .PP
-Например, если вы хотите выполнить какое-то действие над всеми
+Например, если вы хотите выполнить какое\-то действие над всеми
 символическими ссылками текущего каталога, вы можете использовать пункт
 "Критерий панелизации" для запуска следующей команды:
 .PP
 .nf
-find . -type l -print
+find . \-type l \-print
 .fi
 .PP
 После выполнения этой команды в текущей панели будет отображено не
@@ -1425,7 +1425,7 @@
 в нем расположенные.
 .PP
 Если вы хотите вывести на панель все файлы, которые были скачаны с
-вашего FTP-сайта, вы можете использовать следующую команду для
+вашего FTP\-сайта, вы можете использовать следующую команду для
 извлечения нужных имен из файла протокола:
 .PP
 .nf
@@ -1447,7 +1447,7 @@
 Пользуясь диалоговым окном справочника каталогов, вы можете добавить
 новую метку в список или удалить ранее созданную пару метка/каталог. Для
 добавления метки можно также использовать комбинацию клавиш
-.B C-x h,
+.B C\-x h,
 по которой текущий каталог добавляется в справочник каталогов. Программа
 выдаст запрос на ввод метки для этого каталога.
 .PP
@@ -1479,13 +1479,13 @@
 В качестве keyword может стоять одно из следующих слов:
 .TP
 .I shell
-если desc начинается с точки (мета-символы не допускаются), то desc -
+если desc начинается с точки (мета\-символы не допускаются), то desc \-
 это расширение имени файла, т.е. шаблону соответствуют все файлы,
 оканчивающиеся на desc.  Пример: .tar будет означать *.tar.  Если desc
 не начинается с точки, то desc трактуется как полное имя файла.
 .TP
 .I regex
-desc - регулярное выражение. Обратите внимание на то, что используется
+desc \- регулярное выражение. Обратите внимание на то, что используется
 библиотека GNU regex, в силу чего "\\|" соответствует просто символу
 "|", а "|" имеет специальное значение (логическое ИЛИ). Аналогично,
 вместо скобок "(" и ")" надо использовать "\\(" и "\\)".
@@ -1540,7 +1540,7 @@
 из Midnight Commander (такими заданиями могут быть только операции
 копирования и перемещения файлов). Используя этот пункт меню или
 "горячие" клавиши
-.B C-x j,
+.B C\-x j,
 вы можете остановить, возобновить или снять любое из фоновых заданий.
 .\"NODE "    Menu File Edit"
 .SH "    Редактирование файлов меню"
@@ -1569,7 +1569,7 @@
 соответствующего пункта меню. Эти строки должны следовать за строкой
 названия и их может быть несколько.
 .PP
-Когда пользователь выбирает какой-то пункт меню, соответствующие
+Когда пользователь выбирает какой\-то пункт меню, соответствующие
 командные строки копируются во временный файл во временном каталоге
 (обычно /usr/tmp) и затем этот файл запускается на выполнение. Это
 позволяет пользователю вставлять в команды меню любые конструкции,
@@ -1584,30 +1584,30 @@
 .PP
 .nf
 A	Dump the currently selected file
-	od -c %f
+	od \-c %f
 
 B	Edit a bug report and send it to root
-	I=`mktemp ${MC_TMPDIR:-/tmp}/mail.XXXXXX` || exit 1
+	I=`mktemp ${MC_TMPDIR:\-/tmp}/mail.XXXXXX` || exit 1
 	vi $I
-	mail -s "Midnight Commander bug" root < $I
-	rm -f $I
+	mail \-s "Midnight Commander bug" root < $I
+	rm \-f $I
 
 M	Read mail
-	emacs -f rmail
+	emacs \-f rmail
 
 N	Read Usenet news
-	emacs -f gnus
+	emacs \-f gnus
 
 H	Call the info hypertext browser
 	info
 
 J	Copy current directory to other panel recursively
-	tar cf - . | (cd %D && tar xvpf -)
+	tar cf \- . | (cd %D && tar xvpf \-)
 
 K	Make a release of the current subdirectory
-	echo -n "Name of distribution file: "
+	echo \-n "Name of distribution file: "
 	read tar
-	ln -s %d `dirname %d`/$tar
+	ln \-s %d `dirname %d`/$tar
 	cd ..
 	tar cvhf ${tar}.tar $tar
 
@@ -1624,11 +1624,11 @@
 выполняется.
 .PP
 .nf
-Синтаксис строки условия:   = <sub-cond>
-                или:        = <sub-cond> | <sub-cond> ...
-                или:        = <sub-cond> & <sub-cond> ...
+Синтаксис строки условия:   = <sub\-cond>
+                или:        = <sub\-cond> | <sub\-cond> ...
+                или:        = <sub\-cond> & <sub\-cond> ...
 
-где <sub-condition> может принимать следующие значения:
+где <sub\-condition> может принимать следующие значения:
 
   y <pattern>     синтаксис текущего файла соответствует шаблону.
                   Только для меню редактора.
@@ -1643,10 +1643,10 @@
   T <type>        тип файла в пассивной панели
                   соответствует <type>?
   x <filename>    filename является исполнимым?
-  ! <sub-cond>    логическое отрицание условия <sub-cond>
+  ! <sub\-cond>    логическое отрицание условия <sub\-cond>
 .fi
 .PP
-Шаблон (pattern) - это обычный шаблон оболочки (shell pattern) или
+Шаблон (pattern) \- это обычный шаблон оболочки (shell pattern) или
 регулярное выражение, в соответствии с опцией "shell patterns". Вы
 можете задать значение опции "shell patterns" путем записи строки
 "shell_patterns=x" в самую первую строку файла меню (здесь "x" равно 0
@@ -1659,8 +1659,8 @@
   r     обычный файл (regular file)
   d     каталог
   l     ссылка (link)
-  c     байт-ориентированное устройство (char special)
-  b     блок-ориентированное устройство (block special)
+  c     байт\-ориентированное устройство (char special)
+  b     блок\-ориентированное устройство (block special)
   f     fifo
   s     socket
   x     исполняемый файл
@@ -1690,23 +1690,23 @@
 .PP
 .nf
 = f *.tar.gz | f *.tgz & t n
-L	List the contents of a compressed tar-archive
-	gzip -cd %f | tar xvf -
+L	List the contents of a compressed tar\-archive
+	gzip \-cd %f | tar xvf \-
 .fi
 .PP
 .B Условия включения пункта в меню (Addition Conditions)
 .PP
-Если строка условия начинается с '+' (или '+?') вместо '=' (или '=?') -
+Если строка условия начинается с '+' (или '+?') вместо '=' (или '=?') \-
 это условие трактуется как условие включения данного пункта в меню. Если
 условие истинно, пункт включается в отображаемое на экране меню, если
-условие не выполняется - не включается.
+условие не выполняется \- не включается.
 .PP
 Вы можете использовать одно и то же условие и как условие выполнения и
 как условие включения пункта в меню, если начнете строку условия с '+='
 или '=+' (если надо отображать ход вычисления, то с '+=?' или '=+?').
 Если вы хотите использовать два разных условия, одно для включения в
-меню, а другое - для выполнения, надо предпослать пункту меню две строки
-условий, одну - начинающуюся с '+', и другую - начинающуюся с '='.
+меню, а другое \- для выполнения, надо предпослать пункту меню две строки
+условий, одну \- начинающуюся с '+', и другую \- начинающуюся с '='.
 .\"NODE "  Options Menu"
 .SH "Меню 'Настройки' ('Options')"
 Программа Midnight Commander имеет ряд установок (опций), каждая из
@@ -1763,7 +1763,7 @@
 .\"LINK2"
 Сохранить настройки (Save setup)
 .\"Save Setup"
-введенные значения параметров сохраняются в ini-файле программы.
+введенные значения параметров сохраняются в ini\-файле программы.
 .\"NODE "    Configuration"
 .SH "    Конфигурация"
 Параметры конфигурации, задаваемые в этом окне, делятся на три группы:
@@ -1773,11 +1773,11 @@
 .PP
 .I Показывать резервные (Backup) файлы.
 По умолчанию программа Midnight Commander не показывает файлы, имена
-которых заканчиваются на '~' (подобно ключу -B команды ls в GNU).
+которых заканчиваются на '~' (подобно ключу \-B команды ls в GNU).
 .PP
 .I Показывать скрытые файлы.
 По умолчанию Midnight Commander показывает все файлы, имена которых
-начинаются точкой (как ls -a).
+начинаются точкой (как ls \-a).
 .PP
 .I Отметка перемещает курсор.
 Когда вы отмечаете файл (клавишей
@@ -1797,19 +1797,19 @@
 .I Смешивать файлы/каталоги.
 Если эта опция включена, имена файлов и каталогов отображаются
 вперемежку. Если опция отключена, каталоги (и ссылки на каталоги)
-показываются в начале списка, а имена файлов - после имен всех
+показываются в начале списка, а имена файлов \- после имен всех
 каталогов.
 .PP
 .I Быстрая загрузка каталога.
 По умолчанию эта опция выключена. Если вы активизируете ее, Midnight
 Commander будет использовать для вывода содержимого каталога следующий
 трюк: содержимое каталога перечитывается только в том случае, если
-изменилась запись в i-node каталога, то есть если в каталоге создавались
-или удалялись файлы; если изменялись только записи в i-node файлов
+изменилась запись в i\-node каталога, то есть если в каталоге создавались
+или удалялись файлы; если изменялись только записи в i\-node файлов
 каталога (изменялся размер файла, режим доступа или владелец и т.п.)
 содержимое панели не обновляется. В этом случае (если опция включена) вы
 должны обновлять список файлов вручную (клавишами
-.BR C-r ).
+.BR C\-r ).
 .PP
 .B Пауза после исполнения
 .PP
@@ -1820,12 +1820,12 @@
 .I <Никогда>
 Это значит, что вы не хотите видеть вывод команды. На консоли Linux или
 FreeBSD или при использовании xterm вы можете просмотреть этот вывод,
-нажав C-o.
+нажав C\-o.
 .IP
 .I <На "тупых" терминалах>
 Пауза будет создаваться на терминалах, которые не способны обеспечить
 показ вывода последней из выполнявшихся команд (это любые терминалы,
-отличные от xterm или Linux-консоли).
+отличные от xterm или Linux\-консоли).
 .IP
 .I <Всегда>
 Программа обеспечит паузу после выполнения любой команды.
@@ -1849,14 +1849,14 @@
 Если эта опция включена, Midnight Commander перед выполнением операций
 копирования, перемещения и удаления подсчитывает общее число
 обрабатываемых файлов и их суммарный размер и показывает ход выполнения
-операции над этими файлами в виде диаграммы-полоски (правда, это слегка
+операции над этими файлами в виде диаграммы\-полоски (правда, это слегка
 замедляет выполнение операций). Эта опция не работает, если отключена
 опция
 .I Детали операций.
 .PP
 .I Образцы в стиле shell (Shell Patterns).
 По умолчанию команды Select, Unselect и Filter используют регулярные
-выражения, которые строятся по правилам, действующим в оболочке (shell-
+выражения, которые строятся по правилам, действующим в оболочке (shell\-
 like regular expressions). Для того, чтобы достичь такого эффекта,
 выполняются следующие преобразования: '*' заменяется на '.*' (ноль или
 больше символов); '?' заменяется на '.' (в точности один символ) и '.'
@@ -1904,25 +1904,25 @@
 Завершение ввода
 .\"Completion"
 при нажатии на клавиши
-.B M-Tab,
+.B M\-Tab,
 пытаясь угадать окончание вводимой команды. По умолчанию при первом
 нажатии
-.B M-Tab
+.B M\-Tab
 он ищет все возможные варианты завершения и, если завершения
 неоднозначны (имеется много разных вариантов), то только издает звуковой
 сигнал. При втором нажатии
-.B M-Tab
+.B M\-Tab
 отображаются все возможные завершения. Если вы хотите видеть все
 возможные варианты завершения после первого нажатия
-.B M-Tab,
+.B M\-Tab,
 включите эту опцию.
 .PP
 .I Вращающийся индикатор
 Если эта опция включена, Midnight Commander отображает в верхнем правом
-углу вращающуюся черточку, как индикатор того, что выполняется какое-то
+углу вращающуюся черточку, как индикатор того, что выполняется какое\-то
 задание (операция).
 .PP
-.I Навигация в стиле lynx (Lynx-like motion).
+.I Навигация в стиле lynx (Lynx\-like motion).
 Если эта опция включена, вы имеете возможность использовать клавиши
 "стрелка вправо" для перехода в подсвеченный в данный момент каталог и
 "стрелка влево" для перехода в родительский по отношению к текущему
@@ -1958,7 +1958,7 @@
 Диалоговое окно "Внешний вид" дает вам возможность изменить некоторые
 параметры отображения на экране главного окна программы Midnight
 Commander. Вы можете вывести на экран или отменить вывод строки главного
-меню, командной строки, строки подсказок, мини-статуса, строки с
+меню, командной строки, строки подсказок, мини\-статуса, строки с
 подсказкой по функциональным клавишам. На консоли Linux или FreeBSD
 можно задать число строк, которые будут оставлены для отображения вывода
 команды.
@@ -1998,7 +1998,7 @@
 вызываемое при нажатии клавиши F9 меню будет отображаться постоянно.
 .PP
 Если включена опция
-.IR Мини-статус ,
+.IR Мини\-статус ,
 в нижней части каждой панели выводится строка информации о выделенном
 подсветкой файле или каталоге каждой панели.
 .PP
@@ -2035,10 +2035,10 @@
 .\"NODE "    Display bits"
 .SH "    Биты символов..."
 Этот пункт меню используется для задания диапазона отображаемых на
-дисплее символов. Если терминал поддерживает только 7-битный вывод, то
-нужно выбрать установку "7 бит". Выбрав "ISO-8859-1" вы получите все
-символы из таблицы ISO-8859-1, а выбор установки "Полный 8-битный вывод"
-имеет смысл на тех терминалах, которые могут отображать все 8-битовые
+дисплее символов. Если терминал поддерживает только 7\-битный вывод, то
+нужно выбрать установку "7 бит". Выбрав "ISO\-8859\-1" вы получите все
+символы из таблицы ISO\-8859\-1, а выбор установки "Полный 8\-битный вывод"
+имеет смысл на тех терминалах, которые могут отображать все 8\-битовые
 символы.
 .\"NODE "    Learn keys"
 .SH "    Распознавание клавиш..."
@@ -2048,7 +2048,7 @@
 .\"Learn keys"
 вызывает диалоговое окно, в котором вы можете протестировать работу
 некоторых клавиш
-.B (F1-F20, Home, End),
+.B (F1\-F20, Home, End),
 которые работают не на всех типах терминалов.
 .PP
 В диалоговом окне появляется таблица с названиями клавиш, которые могут
@@ -2074,7 +2074,7 @@
 .B Tab
 работает всегда.
 .PP
-Если какая-то клавиша не работает, то после нажатия на нее пометка OK не
+Если какая\-то клавиша не работает, то после нажатия на нее пометка OK не
 появится. В таком случае можно связать ту реакцию системы, которая
 должна была последовать за нажатием неработающей (отсутствующей)
 клавиши, с нажатием другой клавиши или комбинации клавиш. Для этого надо
@@ -2096,7 +2096,7 @@
 Когда закончите тестирование и настройку всех клавиш, вы можете
 сохранить эти настройки в секции [terminal:TERM] вашего файла
 .B ~/.mc/ini
-(где TERM - название используемого терминала) либо отказаться от
+(где TERM \- название используемого терминала) либо отказаться от
 запоминания изменений. Если все клавиши работают нормально, то
 сохранение, конечно, не требуется.
 .\"NODE "    Virtual FS"
@@ -2113,7 +2113,7 @@
 серверов).
 .PP
 Кроме того, для получения доступа к содержимому сжатых архивов
-(например, сжатых tar-файлов) Midnight Commander создает временные файлы
+(например, сжатых tar\-файлов) Midnight Commander создает временные файлы
 на жестком диске.
 .PP
 Поскольку в обоих этих случаях расходуются ресурсы компьютера, у вас
@@ -2123,20 +2123,20 @@
 .PP
 Файловая система архиватора tar очень разумно организует работу с
 архивными файлами: она сохраняет на диске структуру каталогов архива и,
-когда возникает необходимость обращения к информации из tar-файла,
+когда возникает необходимость обращения к информации из tar\-файла,
 использует этот список для извлечения нужного файла.
 .PP
-В общем случае tar-файлы обычно хранятся в сжатом виде (простые
-tar-файлы - вымирающее явление), и в силу природы таких файлов
-(структура подкаталогов для tar-файлов недоступна для мгновенной
+В общем случае tar\-файлы обычно хранятся в сжатом виде (простые
+tar\-файлы \- вымирающее явление), и в силу природы таких файлов
+(структура подкаталогов для tar\-файлов недоступна для мгновенной
 загрузки/отображения), файловая система tar должна разархивировать файлы
 на диск во временный каталог и затем представить обращение к
-разархивированным файлам как обращение к tar-файлу.
+разархивированным файлам как обращение к tar\-файлу.
 .PP
 А поскольку мы все любим быстро пролистывать файлы и хранить их в виде
-tar-архивов, стало самой обычной ситуацией когда сначала просматривают
+tar\-архивов, стало самой обычной ситуацией когда сначала просматривают
 такой файл, выходят из него, а потом обращаются к нему снова. Поскольку
-разархивация - операция медленная, Midnight Commander кеширует
+разархивация \- операция медленная, Midnight Commander кеширует
 информацию в памяти на некоторое время. По истечении этого интервала
 времени все ресурсы, выделенные этой файловой системе, будут
 освобождены. По умолчанию величина этого интервала устанавливается
@@ -2146,17 +2146,17 @@
 .\"Tar File System"
 .PP
 Файловая система FTP сохраняет список каталогов, полученный с
-FTP-сервера, в памяти (в кеше). Время, в течение которого этот список
+FTP\-сервера, в памяти (в кеше). Время, в течение которого этот список
 хранится в кеше, задается опцией
-.I Тайм-аут кеша каталога FTP.
+.I Тайм\-аут кеша каталога FTP.
 Слишком маленькое значение этого параметра будет замедлять все операции
 с файловой системой ftp, потому что каждое обращение к ней будет
 означать реальное обращение к удаленному ftp серверу.
 .PP
-Кроме того, в том же диалоговом окне можно задать имя прокси-сервера
+Кроме того, в том же диалоговом окне можно задать имя прокси\-сервера
 (proxy host), который будет использоваться для перекачки файлов по ftp и
 сконфигурировать Midnight Commander таким образом, чтобы всегда работать
-через прокси-сервер.
+через прокси\-сервер.
 .PP
 .I Пароль анонимного FTP:
 \- пароль, используемый когда вы регистрируетесь как анонимный пользователь
@@ -2185,7 +2185,7 @@
 .BR mc.ini .
 Этот файл располагается в каталоге, путь к которому можно определить,
 запустив команду
-.BR "mc -f" .
+.BR "mc \-f" .
 (это можно сделать даже из командной строки MC).  Если и этого файла не
 существует,
 .I MC
@@ -2213,7 +2213,7 @@
 .PP
 Значения некоторых параметров не могут быть изменены через пункты меню.
 Для изменения значений этих параметров вы должны отредактировать файл
-настроек с помощью какой-либо программы- редактора. Подробнее об этих
+настроек с помощью какой\-либо программы\- редактора. Подробнее об этих
 параметрах смотрите в разделе
 .\"LINK2"
 .I Специальные установки\&.
@@ -2253,13 +2253,13 @@
 вы добавите после тильды имя пользователя, эта конструкция будет
 заменена на имя домашнего каталога указанного пользователя.
 .PP
-Например, ~guest - это домашний каталог пользователя guest, в то время
-как ~/guest - это каталог guest в вашем домашнем каталоге.
+Например, ~guest \- это домашний каталог пользователя guest, в то время
+как ~/guest \- это каталог guest в вашем домашнем каталоге.
 .PP
 .I Предыдущий каталог
-вы можете перейти в предыдущий каталог, используя специальное имя '-',
+вы можете перейти в предыдущий каталог, используя специальное имя '\-',
 то есть выполнив команду:
-.B cd -
+.B cd \-
 .PP
 .I Каталоги в CDPATH
 Если каталог, указанный команде
@@ -2380,13 +2380,13 @@
 того, чтобы отказаться от ввода. Этот макрос пока не работает в
 командной строке.
 .TP
-.I "%var{ENV-VAR:default}"
-Этот макрос будет заменяться на значение переменной окружения ENV-VAR,
+.I "%var{ENV\-VAR:default}"
+Этот макрос будет заменяться на значение переменной окружения ENV\-VAR,
 если она установлена, в противном случае будет использовано значение
-default. Это подобно конструкции ${VAR-def} в оболочке Борна.
+default. Это подобно конструкции ${VAR\-def} в оболочке Борна.
 .\"NODE "  The subshell support"
 .SH "  Поддержка subshell"
-Поддержка subshell - это опция компиляции, которая работает с оболочками
+Поддержка subshell \- это опция компиляции, которая работает с оболочками
 bash, tcsh и zsh.
 .PP
 Если поддержка subshell включена, то Midnight Commander будет порождать
@@ -2394,7 +2394,7 @@
 переменной окружения
 .B SHELL
 или, если эта переменная не определена, в файле /etc/passwd) и запускать
-ее в псевдо-терминале, вместо того, чтобы каждый раз вызывать новую
+ее в псевдо\-терминале, вместо того, чтобы каждый раз вызывать новую
 оболочку для выполнения команды. Введенная команда передается subshell.
 Это позволяет изменять значения переменных окружения, использовать
 функции shell и определять алиасы (aliases), которые будут действовать
@@ -2414,7 +2414,7 @@
 .PP
 Во время выполнения приложения в subshell можно в любой момент нажать
 комбинацию клавиш
-.B C-o
+.B C\-o
 для переключения в Midnight Commander (выполняемая команда или
 приложение будут остановлены). Если вы прервали таким образом исполнение
 команды, вы не сможете запустить другую внешнюю команду, пока не
@@ -2433,12 +2433,12 @@
 .\"NODE "Chmod"
 .SH "Права доступа (Chmod)"
 Диалоговое окно "Права доступа" предоставляет удобный интерфейс к
-команде chmod - команде изменения атрибутов доступа к файлам и
+команде chmod \- команде изменения атрибутов доступа к файлам и
 каталогам. Доступ к этому окну можно получить по "горячей" комбинации
 клавиш
-.BR "C-x c" .
+.BR "C\-x c" .
 .PP
-Диалоговое окно "Права доступа" состоит из двух частей -
+Диалоговое окно "Права доступа" состоит из двух частей \-
 .I Права доступа
 и
 .IR Файл .
@@ -2456,7 +2456,7 @@
 пробела.
 .PP
 Для перемещения подсветки по названиям атрибутов можно использовать
-.I клавиши-стрелки
+.I клавиши\-стрелки
 или клавишу
 .IR Tab .
 Названия изменяемых атрибутов помечаются звездочкой и предполагаемое
@@ -2509,9 +2509,9 @@
 .\"NODE "Chown"
 .SH "Владелец/группа (Chown)"
 Пункт меню "Владелец/группа" используется для того, чтобы изменить
-владельца файла или группу, которая имеет какие-то права на файл.
+владельца файла или группу, которая имеет какие\-то права на файл.
 "Горячая" клавиша для вызова этого пункта меню \-
-.B C-x o.
+.B C\-x o.
 .\"NODE "Advanced Chown"
 .SH "Расширенная команда Chown"
 Выбрав пункт меню "Расширенная команда Chown", вы увидите окно,
@@ -2531,7 +2531,7 @@
 файлов, Midnight Commander отображает на экране диалоговое окно, в
 котором показано, какой(ие) файл(ы) в данный момент обрабатывается и как
 идет процесс обработки. Для отображения процесса обработки на экран
-выводится до трех диаграмм-полосок (progress bars). Первая (file bar)
+выводится до трех диаграмм\-полосок (progress bars). Первая (file bar)
 показывает, какая часть текущего файла обработана (например,
 скопирована). Вторая (count bar) показывает, какая доля помеченных файлов
 обработана на текущий момент. Третья (bytes bar) показывает долю
@@ -2556,7 +2556,7 @@
 .PP
 Окно подтверждения перезаписи появляется тогда, когда вы пытаетесь
 переписать существующий файл. В этом окне отображаются время создания и
-размер файла-источника и файла-приемника (то есть перезаписываемого
+размер файла\-источника и файла\-приемника (то есть перезаписываемого
 файла, который фактически будет уничтожен). Ниже выведены два вопроса.
 На первый вопрос ("Переписать этот файл?") предлагается три варианта
 ответа:
@@ -2565,7 +2565,7 @@
 .PP
 отказаться, то есть пропустить перезапись текущего файла (кнопка <Нет>);
 .PP
-дописать содержимое файла-источника в конец файла-приемника (кнопка
+дописать содержимое файла\-источника в конец файла\-приемника (кнопка
 <дописать в Конец>).
 .PP
 Выбор одного из вариантов ответа на второй вопрос ("Переписать все
@@ -2578,10 +2578,10 @@
 .TP
 .I <Устаревшие>
 перезапишутся (затрутся) только файлы, которые созданы раньше
-файла-источника;
+файла\-источника;
 .TP
 .I <ни Одного>
-не перезаписывать файлы (но если не существует файла-приемника, то
+не перезаписывать файлы (но если не существует файла\-приемника, то
 копирование источника будет произведено);
 .TP
 .I <Различающиеся по длине>
@@ -2593,7 +2593,7 @@
 пытаетесь удалить непустой каталог. По кнопке <Да> каталог будет удален
 вместе со всеми файлами, <Нет> означает отказ от удаления каталога,
 <Все> надо выбирать, если вы отметили группу подкаталогов для удаления и
-уверены в своем выборе, <ни Одного> - чтобы пропустить все непустые
+уверены в своем выборе, <ни Одного> \- чтобы пропустить все непустые
 каталоги из числа помеченных, <Прервать> означает отказ от выполнения
 операции удаления. Выбирайте <Да> или <Все> только в том случае, когда
 вы действительно уверены, что хотите удалить каталог со всеми вложенными
@@ -2606,30 +2606,30 @@
 .SH "Маски файлов для операций копирования/переименования"
 При выполнении операций копирования и перемещения (или переименования)
 файлов вы имеете возможность изменить имена копируемых или перемещаемых
-файлов. Для этого вы должны задать маску для имен файлов-источников и
-маску для имен файлов, которые будут созданы (файлы-приемники). Обычно
+файлов. Для этого вы должны задать маску для имен файлов\-источников и
+маску для имен файлов, которые будут созданы (файлы\-приемники). Обычно
 эта вторая маска представляет собой несколько символов замены
 (wildcards) в конце строки, определяющей место назначение создаваемых
 файлов. Все файлы, удовлетворяющие маске источника, будут переименованы
 (скопированы или перемещены с новыми именами) в соответствии с маской
-файла-приемника. Если имеются помеченные файлы, то копируются
+файла\-приемника. Если имеются помеченные файлы, то копируются
 (перемещаются) только помеченные файлы, удовлетворяющие заданной маске
-для файлов-источников.
+для файлов\-источников.
 .PP
 Есть еще несколько опций, которые влияют на выполнение операций
 копирования/перемещения файлов, и которые устанавливаются в том же окне
 запроса, где задаются маски имен файлов:
 .PP
 "Разыменовывать ссылки" (Follow links) определяет, будут ли при
-копировании жестких или символических ссылок в каталоге- приемнике (и
+копировании жестких или символических ссылок в каталоге\- приемнике (и
 рекурсивно в подкаталогах) создаваться такие же ссылки, или будут
 копироваться файлы (и подкаталоги), на которые эти ссылки указывают.
 .PP
 "Внутрь каталога, если есть" (Dive into subdirs) определяет, что делать,
-если в каталоге-приемнике уже существует подкаталог, имя которого
+если в каталоге\-приемнике уже существует подкаталог, имя которого
 совпадает с именем файла (каталога), который копируется (источника). По
-умолчанию (опция отключена) содержимое каталога-источника копируется в
-каталог-приемник. Если опция включена, то в каталоге приемнике будет
+умолчанию (опция отключена) содержимое каталога\-источника копируется в
+каталог\-приемник. Если опция включена, то в каталоге приемнике будет
 создан новый подкаталог с тем же именем, в который и будет
 осуществляться копирование. Лучше показать это на примере:
 .PP
@@ -2657,18 +2657,18 @@
 группе символов замены в маске источника, второй символ '*'
 соответствует второй группе и т.д. Аналогично, символ замены \&'\\1'
 соответствует первой группе символов замены в маске источника, символ
-\&'\\2' - второй группе и т.д. до \&'\\9'. Символ \&'\\0' соответствует
-целому имени файла-источника.
+\&'\\2' \- второй группе и т.д. до \&'\\9'. Символ \&'\\0' соответствует
+целому имени файла\-источника.
 .PP
 Два примера:
 .PP
-Если маска источника "*.tar.gz", а приемник - "/bla/*.tgz" и имя
-копируемого файла - "foo.tar.gz", копия будет называться "foo.tgz" и
+Если маска источника "*.tar.gz", а приемник \- "/bla/*.tgz" и имя
+копируемого файла \- "foo.tar.gz", копия будет называться "foo.tgz" и
 будет находиться в каталоге "/bla".
 .PP
 Предположим вы хотите поменять местами имя и расширение файла, так чтобы
 "file.c" стал файлом "c.file". Маска источника для этого должна иметь
-вид "*.*", а маска приемника - "\\2.\\1".
+вид "*.*", а маска приемника \- "\\2.\\1".
 .PP
 Опция
 .B "Образцы в стиле shell"
@@ -2679,7 +2679,7 @@
 выключена, MC не осуществляет автоматической группировки. Для указания
 групп символов в маске источника, которые будут соответствовать символам
 замены в маске приемника, вы должны в этом случае использовать скобки
-'\\(...\\)'. Этот способ более гибкий, но требует больше усилий при
+\&'\\(...\\)'. Этот способ более гибкий, но требует больше усилий при
 вводе.
 .PP
 Два примера:
@@ -2690,7 +2690,7 @@
 .PP
 Предположим, что вы хотите поменять местами имя файла и его расширение,
 так чтобы имена вида "file.c" приняли вид "c.file". Маска источника для
-этого - "^\\(.*\\)\\.\\(.*\\)$", а маска приемника - "\\2.\\1".
+этого \- "^\\(.*\\)\\.\\(.*\\)$", а маска приемника \- "\\2.\\1".
 .PP
 .B "Преобразование регистра"
 .PP
@@ -2705,12 +2705,12 @@
 или конца имени файла.
 .PP
 Применение '\\u' и '\\l' обеспечивает более широкие возможности, чем
-'\\U' и '\\L'.
+\&'\\U' и '\\L'.
 .PP
 Например, если маска источника есть '*' (опция "Образцы в стиле shell"
 включена) или '^\\(.*\\)$' (опция "Образцы в стиле shell" выключена), а
 маска приемника есть '\\L\\u*', имена файлов будут преобразованы таким
-образом, что первые буквы имени будут заглавными, а все остальные -
+образом, что первые буквы имени будут заглавными, а все остальные \-
 строчными.
 .PP
 Символ '\\' в масках используется для отмены специальной интерпретации
@@ -2759,10 +2759,10 @@
 шаблоне поиска как ту, так и другую форму представления, например:
 .PP
 .nf
-"String" -1 0xBB 012 "more text"
+"String" \-1 0xBB 012 "more text"
 .fi
 .PP
-Обратите внимание, что 012 является восьмеричным числом, -1
+Обратите внимание, что 012 является восьмеричным числом, \-1
 преобразовывается в 0xFF, а текст между кавычками и константами
 игнорируется.
 .PP
@@ -2800,14 +2800,14 @@
 Вызов диалогового окна для задания шаблона поиска для обычного и
 шестнадцатеричного поиска.
 .PP
-.B C-s, F17, n
+.B C\-s, F17, n
 Если шаблон поиска еще не задан, вызывается диалоговое окно для его
 задания, иначе повторяется поиск (дальше по тексту) в соответствии с
 ранее определенным шаблоном.
 .PP
-.B C-r
+.B C\-r
 То же, что и
-.BR C-s ,
+.BR C\-s ,
 но поиск производится в обратном направлении.
 .PP
 .B n
@@ -2831,10 +2831,10 @@
 .B F3, F10, Esc, q
 Выйти из встроенной программы просмотра.
 .PP
-.B PageDown, пробел, C-v
+.B PageDown, пробел, C\-v
 Переместиться на страницу вперед.
 .PP
-.B PageUp, M-v, backspace.
+.B PageUp, M\-v, backspace.
 Переместиться на страницу назад.
 .PP
 .B стрелка вниз
@@ -2843,16 +2843,16 @@
 .B стрелка вверх
 Переместиться на одну строку назад.
 .PP
-.B C-l
+.B C\-l
 Обновить изображение на экране.
 .PP
-.B C-o
+.B C\-o
 Переключиться в subshell и показать окно команд.
 .PP
 .B !
 Запустить новый экземпляр оболочки в текущем каталоге (временный выход в
 shell, возврат в программу просмотра по
-.BR C-d ).
+.BR C\-d ).
 .PP
 .B "[n] m"
 Установить метку с номером n (кавычки не вводятся).
@@ -2860,13 +2860,13 @@
 .B "[n] r"
 Переместиться к метке с номером n.
 .PP
-.B C-f
+.B C\-f
 Перейти к просмотру следующего файла текущего каталога.
 .PP
-.B C-b
+.B C\-b
 Перейти к просмотру предыдущего файла в текущем каталоге.
 .PP
-.B M-r
+.B M\-r
 Переключение режима отображения линейки (вверху/внизу/отключена).
 .PP
 Можно настроить режимы отображения файлов встроенной программой
@@ -2890,8 +2890,8 @@
 .I (key for key undo);
 выпадающие меню; вставка файлов; макроопределения; поиск и замена по
 регулярным выражениям; выделение текста по комбинации клавиш
-shift-стрелки в стиле MSW-MAC (только для linux-консоли);
-переключение между режимами вставки-замены символа; а
+shift\-стрелки в стиле MSW\-MAC (только для linux\-консоли);
+переключение между режимами вставки\-замены символа; а
 также операция обработки блоков текста командами оболочки (an option to
 pipe text blocks through shell commands like indent).
 .PP
@@ -2899,15 +2899,15 @@
 узнать, какие клавиши вызывают выполнение определенных действий,
 достаточно просмотреть выпадающие меню, которые вызываются нажатием
 клавиши F9 в окне редактора. Не перечисленные в меню комбинации клавиш:
-.B Shift-<клавиши стрелок>
+.B Shift\-<клавиши стрелок>
 выделение блока текста.
-.B Ctrl-Ins
+.B Ctrl\-Ins
 копирует блок в файл
 .BR cooledit.clip .
-.B Shift-Ins
+.B Shift\-Ins
 производит вставку последнего скопированного в cooledit.clip блока в
 позицию курсора.
-.B Shift-Del
+.B Shift\-Del
 удаляет выделенный блок текста, запоминая его в файле
 .BR cooledit.clip .
 По клавише Enter вставляются символы конца строки, причем на следующей
@@ -2916,16 +2916,16 @@
 осуществляется терминальным драйвером мыши.
 .PP
 Для того, чтобы определить макрос, нажмите
-.BR Ctrl-R ,
+.BR Ctrl\-R ,
 после чего введите строки команд, которые должны быть выполнены. После
 завершения ввода команд снова нажмите
-.B Ctrl-R
-и свяжите макрос с какой-нибудь клавишей или комбинацией клавиш, нажав
+.B Ctrl\-R
+и свяжите макрос с какой\-нибудь клавишей или комбинацией клавиш, нажав
 эту клавишу (комбинацию). Макрос будет вызываться нажатием
-.B Ctrl-A
+.B Ctrl\-A
 и назначенной для него клавиши. Макрос можно также вызвать нажатием
 любой из клавиш Meta (Alt), Ctrl, или Esc и назначенной макросу клавиши,
-при условии, что данная комбинация не используется для вызова какой-либо
+при условии, что данная комбинация не используется для вызова какой\-либо
 другой функции. Макрокоманды после определения записываются в файл
 .B .mc/cedit/cooledit.macros
 в вашем домашнем каталоге. Вы можете удалить макрос удалением
@@ -2953,7 +2953,7 @@
 Запомните только, что все основные операции можно выполнить через пункты
 меню, которое вызывается нажатием клавиши
 .B F9
-в окне редактирования. Кроме того, можно прочитать man-страницу по
+в окне редактирования. Кроме того, можно прочитать man\-страницу по
 команде
 .B man mcedit
 или
@@ -2962,7 +2962,7 @@
 .SH "Завершение ввода (Completion)"
 Заставьте Midnight Commander работать на клавиатуре вместо вас!
 .PP
-Завершение ввода - это попытка закончить за вас ввод текста, набранного
+Завершение ввода \- это попытка закончить за вас ввод текста, набранного
 до текущей позиции курсора. MC пытается завершить ввод, трактуя уже
 введенный текст как переменную (если текст начинается с
 .BR $ ),
@@ -2977,7 +2977,7 @@
 категорий, выполняется попытка завершения текста как имени файла.
 .PP
 Завершение имен файлов, пользователей, переменных и машин работает во
-всех строках ввода, завершение команд - только в командной строке. Если
+всех строках ввода, завершение команд \- только в командной строке. Если
 однозначно выполнить завершение ввода невозможно (имеется несколько
 вариантов), MC издает звуковой сигнал и выполняет следующие действия, в
 зависимости от установки опции
@@ -2992,7 +2992,7 @@
 .BR Enter .
 Вы можете также набрать следующую букву, чтобы перейти к подмножеству
 всех возможных завершений. Если вы снова нажмете
-.BR M-Tab ,
+.BR M\-Tab ,
 будет выведено уже сокращенное множество вариантов, причем в строку
 ввода добавляются те символы, которые определяются однозначно (даже если
 полное завершение ввода еще невозможно). Как только неопределенность
@@ -3006,18 +3006,18 @@
 .\"Configuration"
 отключена, окно с вариантами завершения появляется только тогда, когда
 вы нажмете клавиши
-.B M-Tab
+.B M\-Tab
 второй раз, при первом нажатии MC только издает звуковой сигнал.
 .\"NODE "Virtual File System"
 .SH "Виртуальные файловые системы"
 Программа Midnight Commander содержит подпрограммы, обеспечивающие
 доступ к различным файловым системам. Эти подпрограммы (их совокупность
-называется переключателем виртуальных файловых систем - virtual file
-system switch) позволяют Midnight Commander-у манипулировать файлами,
-расположенными на не-Unix-овых файловых системах.
+называется переключателем виртуальных файловых систем \- virtual file
+system switch) позволяют Midnight Commander\-у манипулировать файлами,
+расположенными на не\-Unix\-овых файловых системах.
 .PP
 В настоящее время Midnight Commander обеспечивает поддержку нескольких
-Виртуальных Файловых Систем - ВФС (VFS):
+Виртуальных Файловых Систем \- ВФС (VFS):
 .br
 \- локальной файловой системы, используемой для обычных файловых систем
 Unix;
@@ -3025,8 +3025,8 @@
 \- файловой системы ftpfs, используемой для манипулирования файлами на
 удаленных компьютерах по протоколу FTP;
 .br
-\- файловой системы tarfs, используемой для обработки tar- и сжатых
-tar-файлов;
+\- файловой системы tarfs, используемой для обработки tar\- и сжатых
+tar\-файлов;
 .br
 \- файловой системы undelfs, используемой для восстановления удаленных
 файлов в файловой системе ext2 (файловая система, используемая в Linux
@@ -3072,17 +3072,17 @@
 .\"NODE "  FTP File System"
 .SH "Файловая система ftpfs (FTP File System)"
 Файловая система ftpfs позволяет работать с файлами на удаленных
-компьютерах. Для этого можно использовать команду "FTP-соединение"
+компьютерах. Для этого можно использовать команду "FTP\-соединение"
 (доступную из меню левой и правой панелей) или же непосредственно
 сменить текущий каталог командой cd, задав путь к каталогу следующим
 образом:
 .PP
-.I /#ftp:[!][user[:pass]@]machine[:port][remote-dir]
+.I /#ftp:[!][user[:pass]@]machine[:port][remote\-dir]
 .PP
 Элементы
 .IR user ", " port
 и
-.I remote-dir
+.I remote\-dir
 не обязательны. Если элемент
 .I user
 указан, то Midnight Commander будет пытаться регистрироваться на
@@ -3097,9 +3097,9 @@
 .PP
 .nf
     /#ftp:ftp.nuclecu.unam.mx/linux/local
-    /#ftp:tsx-11.mit.edu/pub/linux/packages
+    /#ftp:tsx\-11.mit.edu/pub/linux/packages
     /#ftp:!behind.firewall.edu/pub
-    /#ftp:guest@remote-host.com:40/pub
+    /#ftp:guest@remote\-host.com:40/pub
     /#ftp:miguel:xxx@server/pub
 .fi
 .PP
@@ -3113,7 +3113,7 @@
 .\"Virtual FS"
 меню "Настройки".
 .PP
-Чтобы не задавать имя прокси-сервера каждый раз, можно поставить отметку
+Чтобы не задавать имя прокси\-сервера каждый раз, можно поставить отметку
 в квадратных скобках возле опции
 .I Всегда использовать FTP прокси
 в диалоговом окне
@@ -3121,20 +3121,20 @@
 Виртуальные ФС...
 .\"Virtual FS"
 меню "Настройки". В таком случае программа всегда будет использовать
-указанный прокси-сервер. При этом (если опция установлена) программа
+указанный прокси\-сервер. При этом (если опция установлена) программа
 делает следующее: считывает из файла
 .B @prefix@/share/mc/mc.no_proxy
 имена локальных машин (если имя начинается с точки, оно считается именем
-домена), и, если заданное при установлении FTP-соединения имя машины
+домена), и, если заданное при установлении FTP\-соединения имя машины
 совпадает с одним из имен, указанных в файле
 .B mc.no_proxy
 без точки, то производит прямое обращение к данной машине.
 .PP
-При подключении к ftp-серверу через фильтрующий пакеты
+При подключении к ftp\-серверу через фильтрующий пакеты
 маршрутизатор (If you are using the ftpfs code with a filtering packet
 router), который не позволяет использовать обычный режим открытия
 файлов, можно заставить программу работать в режиме пассивного
-открытия файла (the passive-open mode). Для этого установите в
+открытия файла (the passive\-open mode). Для этого установите в
 инициализационном файле опцию
 .I ftpfs_use_passive_connections
 в 1.
@@ -3146,25 +3146,25 @@
 Виртуальные ФС...
 .\"Virtual FS"
 меню "Настройки". В силу этого возможен побочный эффект, заключающийся в
-том, что даже если вы сделали какие-то изменения в каталоге, они не
+том, что даже если вы сделали какие\-то изменения в каталоге, они не
 будут отображаться в панели до тех пор, пока вы не обновите содержимое
 панели командой
-.BR C-r .
+.BR C\-r .
 Это не является недоработкой (если вы думаете, что это ошибка,
 поразмыслите над тем, как происходит работа по FTP с файлами,
 находящимися на другой стороне Атлантического океана).
 .\"NODE "  Tar File System"
 .SH "Файловая система tar"
-Файловая система tar обеспечивает доступ по чтению к tar- и сжатым
-tar-файлам, используя команду chdir. Для перехода в каталог,
-представляющий список файлов tar-файла, нужно использовать команду,
+Файловая система tar обеспечивает доступ по чтению к tar\- и сжатым
+tar\-файлам, используя команду chdir. Для перехода в каталог,
+представляющий список файлов tar\-файла, нужно использовать команду,
 имеющую следующий формат:
 .PP
-.I cd ./filename.tar#utar/[dir-inside-tar]
+.I cd ./filename.tar#utar/[dir\-inside\-tar]
 .PP
 Файл mc.ext по умолчанию содержит команды для просмотра
-tar-файлов, то есть обычно для получения списка файлов tar-архива
-достаточно  просто переместить указатель на имя tar-файла и нажать
+tar\-файлов, то есть обычно для получения списка файлов tar\-архива
+достаточно  просто переместить указатель на имя tar\-файла и нажать
 Enter. Чтобы понять, как это достигается, смотрите раздел
 .\"LINK2"
 Файл расширений\&.
@@ -3173,28 +3173,28 @@
 Примеры:
 .PP
 .nf
-    mc-3.0.tar.gz#utar/mc-3.0/vfs
-    /ftp/GCC/gcc-2.7.0.tar#utar
+    mc\-3.0.tar.gz#utar/mc\-3.0/vfs
+    /ftp/GCC/gcc\-2.7.0.tar#utar
 .fi
 .PP
-В последнем примере указан полный путь к tar-архиву.
+В последнем примере указан полный путь к tar\-архиву.
 .\"NODE "  FIle transfer over SHell filesystem"
 .SH "  Файловая система FISH (FIle transfer over SHell)"
-Файловая система fish - это сетевая файловая система, которая позволяет
+Файловая система fish \- это сетевая файловая система, которая позволяет
 работать с файлами на удаленном компьютере так, как если бы они были
 расположены на вашем диске. Для того, чтобы это было возможно, на
-удаленном компьютере должен быть запущен fish-сервер, или
-bash-совместимая оболочка shell.
+удаленном компьютере должен быть запущен fish\-сервер, или
+bash\-совместимая оболочка shell.
 .PP
 Для соединения с удаленным компьютером нужно выполнить команду
 перехода в каталог (chdir), имя которого задается в следующем формате:
 .PP
-.I /#sh:[user@]machine[:options]/[remote-dir]
+.I /#sh:[user@]machine[:options]/[remote\-dir]
 .PP
 Элементы
 .IR user ", " options
 и
-.I remote-dir
+.I remote\-dir
 не обязательны. Если задан элемент
 .IR user ,
 то Midnight Commander будет регистрироваться на удаленный компьютер под
@@ -3205,11 +3205,11 @@
 .I options
 могут использоваться:
 .nf
-  'C' - использовать сжатие и
-  'r' - использовать rsh вместо ssh.
+  'C' \- использовать сжатие и
+  'r' \- использовать rsh вместо ssh.
 .fi
 Если задан элемент
-.IR remote-dir ,
+.IR remote\-dir ,
 то указанный каталог станет текущим после соединения с удаленным
 компьютером.
 .PP
@@ -3232,12 +3232,12 @@
 перехода в каталог, имя которого строится в соответствии со следующим
 форматом:
 .PP
-.I /#mc:[user@]machine[:port][remote-dir]
+.I /#mc:[user@]machine[:port][remote\-dir]
 .PP
 Элементы
 .IR user ", " port
 и
-.I remote-dir
+.I remote\-dir
 не обязательны.
 .PP
 Если задан элемент
@@ -3253,7 +3253,7 @@
 смотрите страницу руководства
 .BR mcserv(8) .
 Если задан элемент
-.IR remote-dir ,
+.IR remote\-dir ,
 то указанный каталог станет текущим после соединения с удаленным
 компьютером.
 .PP
@@ -3279,7 +3279,7 @@
 система.
 .PP
 Например, чтобы восстановить удаленные файлы на втором разделе первого
-SCSI-диска, нужно использовать следующее имя:
+SCSI\-диска, нужно использовать следующее имя:
 .PP
 .nf
     /#undel:sda2
@@ -3299,12 +3299,12 @@
 панелей) или же непосредственно сменить текущий каталог командой cd,
 задав путь к каталогу следующим образом:
 .PP
-.I /#smb:[username@]machine[/service][/remote-dir]
+.I /#smb:[username@]machine[/service][/remote\-dir]
 .PP
 Элементы
 .IR username ", " service
 и
-.I remote-dir
+.I remote\-dir
 необязательны.
 .IR username ", " domain
 и
@@ -3334,10 +3334,10 @@
 не дает возможности включать цветной режим: ncurses использует только
 информацию из базы данных о терминалах. Для того, чтобы узнать, какая
 библиотека была использована при компиляции, дайте команду
-.BR "mc -V" .
+.BR "mc \-V" .
 .PP
 Если программа скомпилирована с использованием менеджера экрана
-.BR S-Lang ,
+.BR S\-Lang ,
 она проверяет значение переменной
 .BR COLORTERM .
 Задание этой переменной дает тот же эффект, что и задание ключа \-c.
@@ -3354,7 +3354,7 @@
 color_terminals=linux,xterm
 .fi
 .nf
-color_terminals=terminal-name1,terminal-name2...
+color_terminals=terminal\-name1,terminal\-name2...
 .fi
 .PP
 Midnight Commander обеспечивает также возможность изменять цвета,
@@ -3423,9 +3423,9 @@
 .I helpnormal
 \- для отображения простого текста,
 .I helpitalic
-\- для вывода текста, выделенного в man-страницах курсивом,
+\- для вывода текста, выделенного в man\-страницах курсивом,
 .I helpbold
-\- для вывода текста, выделенного в man-страницах жирным шрифтом,
+\- для вывода текста, выделенного в man\-страницах жирным шрифтом,
 .I helplink
 \- для выделения неактивных гиперссылок,
 .I helpslink
@@ -3488,13 +3488,13 @@
 переходит в этот каталог. Если этот флаг будет установлен в 1,
 .B MC
 будет спрашивать подтверждение перед тем, как перейти в указанный
-каталог, если в текущем каталоге были помечены какие-либо файлы.
+каталог, если в текущем каталоге были помечены какие\-либо файлы.
 .PP
 .I ftpfs_retry_seconds
 .IP
 Этот параметр задает интервал времени (в секундах), по истечении
 которого Midnight Commander будет предпринимать повторную попытку
-соединиться с ftp-сервером после того, как предыдущая попытка окончилась
+соединиться с ftp\-сервером после того, как предыдущая попытка окончилась
 неудачей. Если задано нулевое значение, программа не будет повторять
 попытку соединиться.
 .PP
@@ -3517,7 +3517,7 @@
 значение этого параметра может привести к тому, что процедура обновления
 экрана начнет вaс раздражать.
 .IP
-По-видимому значение max_dirt_limit, равное 10, обеспечивает наилучший
+По\-видимому значение max_dirt_limit, равное 10, обеспечивает наилучший
 выбор, и именно такое значение устанавливается по умолчанию.
 .PP
 .I mouse_move_pages
@@ -3535,8 +3535,8 @@
 .I old_esc_mode
 .IP
 По умолчанию Midnight Commander трактует нажатие на клавишу ESC как
-действие, предшествующее нажатию какой-то другой клавиши
-(old_esc_mode=0), и совместно обрабатывает комбинацию <Esc-key>. Если
+действие, предшествующее нажатию какой\-то другой клавиши
+(old_esc_mode=0), и совместно обрабатывает комбинацию <Esc\-key>. Если
 установить эту опцию в 1 (old_esc_mode=1), то программа будет ожидать
 нажатия второй клавиши только в течение 1 секунды, и если за это время
 ни одна клавиша не нажата, то ESC интерпретируется как требование отказа
@@ -3544,7 +3544,7 @@
 .PP
 .I only_leading_plus_minus
 .IP
-устанавливает специальный режим обработки символов '+', '-', '*' в
+устанавливает специальный режим обработки символов '+', '\-', '*' в
 командной строке. Эти символы используются для выбора, отмены выбора и
 инвертирования выбора, но выполняют такую функцию только если командная
 строка пуста. В середине командной строки эти символы уже не вызывают
@@ -3565,7 +3565,7 @@
 .IP
 Эта переменная работает только в том случае, когда не включена поддержка
 subshell. Если она установлена, то при нажатии
-.B C-o
+.B C\-o
 для перехода в экран пользователя будет запускаться новый экземпляр
 оболочки.
 .PP
@@ -3576,13 +3576,13 @@
 (курсор) к первому или последнему из имен файлов, отображенных на
 панели, нажатие этих клавиш будет вызывать следующую реакцию:
 .IP
-клавиша Home - перемещение курсора вверх до середины панели, если курсор
+клавиша Home \- перемещение курсора вверх до середины панели, если курсор
 был ниже, в противном случае перемещение на самую верхнюю строку; если
 только курсор уже на самой верхней строке, то перемещение на самую
 первую строку списка файлов.
 .IP
-клавиша End вызывает аналогичную реакцию - смещение вниз до середины
-панели, если курсор был выше; если ниже - то переход на последнюю
+клавиша End вызывает аналогичную реакцию \- смещение вниз до середины
+панели, если курсор был выше; если ниже \- то переход на последнюю
 отображаемую в панели строку; если уже на последней строке, то
 перемещение курсора на самую последнюю строку списка файлов.
 .PP
@@ -3612,7 +3612,7 @@
 .B mc.lib
 в библиотечном каталоге программы Midnight Commander) или в файле
 .B ~/.mc/ini
-секцию с названием "terminal:your-terminal-name", а затем секцию 
+секцию с названием "terminal:your\-terminal\-name", а затем секцию 
 "terminal:general". Каждая строка в этих секциях содержит ключевое
 слово, которое вы хотите определить, за которым следует знак равенства и
 определение ключевого слова. Специальный символ
@@ -3620,14 +3620,14 @@
 используется для задания экранирующих последовательностей, а
 .B ^x
 служит для задания последовательностей вида
-.B control-x
+.B control\-x
 .PP
 В качестве ключевых слов используются:
 .PP
 .nf
-f0 - f20      определяет последовательность символов,
+f0 \- f20      определяет последовательность символов,
               генерируемых при нажатии на функциональные
-              клавиши f0-f20
+              клавиши f0\-f20
 bs            определяет последовательность символов, которая
               будет использоваться вместо клавиши backspace
 home          то же для клавиши Home
@@ -3653,7 +3653,7 @@
 .fi
 .PP
 Операция "завершение ввода" обычно вызывается по клавишам
-.B M-tab,
+.B M\-tab,
 но задав значение ключевого слова
 .I complete
 можно вызывать такую же реакцию системы по другой комбинации клавиш
@@ -3668,7 +3668,7 @@
 Если эта переменная не задана, происходит обращение к так называемому
 библиотечному каталогу программы Midnight Commander. Для того, чтобы
 узнать имя этого каталога, дайте команду
-.BR "mc -f" .
+.BR "mc \-f" .
 .PP
 .I @prefix@/share/mc/mc.hlp
 .IP
@@ -3731,7 +3731,7 @@
 команду
 .IP
 .nf
- find / -type d -print | sort > ~/.mc/Tree
+ find / \-type d \-print | sort > ~/.mc/Tree
 .fi
 .IP
 Обычно не имеет смысла делать это, потому что Midnight Commander
@@ -3746,7 +3746,7 @@
 .\" "LICENSE"
 .SH Лицензия
 Эта программа распространяется в надежде, что она будет полезной, но БЕЗ
-КАКИХ-ЛИБО ГАРАНТИЙ; даже без подразумеваемых гарантий КОММЕРЧЕСКОЙ
+КАКИХ\-ЛИБО ГАРАНТИЙ; даже без подразумеваемых гарантий КОММЕРЧЕСКОЙ
 ЦЕННОСТИ или ПРИГОДНОСТИ ДЛЯ КОНКРЕТНОЙ ЦЕЛИ.  Для получения подробных
 сведений смотрите Универсальную Общественную Лицензию GNU.
 .\"NODE "AVAILABILITY"
@@ -3760,7 +3760,7 @@
 .PP
 .nf
 Страница, посвященная Midnight Commander, в World Wide Web:
-	http://www.midnight-commander.org/
+	http://www.midnight\-commander.org/
 .fi
 .PP
 Данная страница оперативного руководства содержит информацию, актуальную
@@ -3783,27 +3783,27 @@
 Machek (pavel@ucw.cz) and Pavel Roskin (proski@gnu.org) are the
 developers of this package. Alessandro Rubini (rubini@ipvvis.unipv.it)
 has been especially helpful debugging and enhancing the program's mouse
-support, John Davis (davis@space.mit.edu) also made his S-Lang library
+support, John Davis (davis@space.mit.edu) also made his S\-Lang library
 available to us under the GPL and answered my questions about it, and
 the following people have contributed code and many bug fixes (in
 alphabetical order):
 .PP
 Adam Tla/lka (atlka@sunrise.pg.gda.pl), alex@bcs.zp.ua (Alex I.
 Tkachenko), Antonio Palama, DOS port (palama@posso.dm.unipi.it), Erwin
-van Eijk (wabbit@corner.iaf.nl), Gerd Knorr (kraxel@cs.tu-berlin.de),
-Jean-Daniel Luiset (luiset@cih.hcuge.ch), Jon Stevens
+van Eijk (wabbit@corner.iaf.nl), Gerd Knorr (kraxel@cs.tu\-berlin.de),
+Jean\-Daniel Luiset (luiset@cih.hcuge.ch), Jon Stevens
 (root@dolphin.csudh.edu), Juan Francisco Grigera, Win32 port
-(j-grigera@usa.net), Juan Jose Ciarlante (jjciarla@raiz.uncu.edu.ar),
+(j\-grigera@usa.net), Juan Jose Ciarlante (jjciarla@raiz.uncu.edu.ar),
 Ilya Rybkin (rybkin@rouge.phys.lsu.edu), Marcelo Roccasalva
 (mfroccas@raiz.uncu.edu.ar), Massimo Fontanelli (MC8737@mclink.it),
 Sergey Ya. Korshunoff (root@seyko.msk.su), Thomas Pundt
-(pundtt@math.uni-muenster.de), Timur Bakeyev
+(pundtt@math.uni\-muenster.de), Timur Bakeyev
 (timur@goff.comtat.kazan.su), Tomasz Cholewo
 (tjchol01@mecca.spd.louisville.edu), Torben Fjerdingstad
-(torben.fjerdingstad@uni-c.dk), Vadim Sinolitis (vvs@nsrd.npi.msu.su)
+(torben.fjerdingstad@uni\-c.dk), Vadim Sinolitis (vvs@nsrd.npi.msu.su)
 and Wim Osterholt (wim@djo.wtm.tudelft.nl).
 .PP
-Перевод текста подсказки на русский язык - Костромин В.А., 1999г.
+Перевод текста подсказки на русский язык \- Костромин В.А., 1999г.
 Исправления, адаптация к текущей версии и конвертация в формат mandoc \-
 Самойлов А.В. <sav@bcs.zp.ua>, 2002 г.
 .\"NODE "BUGS"
@@ -3811,9 +3811,9 @@
 Известные авторам недоработки программы перечислены в файле TODO,
 который можно найти в составе поставляемого пакета.
 .PP
-Если вы обнаружили в программе какие-то недостатки или недоработки,
-пришлите, пожалуйста, ваши замечания по e-mail на адрес:
-.IR mc-devel@gnome.org .
+Если вы обнаружили в программе какие\-то недостатки или недоработки,
+пришлите, пожалуйста, ваши замечания по e\-mail на адрес:
+.IR mc\-devel@gnome.org .
 .PP
 Дайте в письме подробное описание обнаруженных недостатков (и/или ваших
 предложений по усовершенствованию программы), сообщите версию программы
